src/animation/AnimationUtils.js

```diff
@@ -158,9 +158,55 @@ var AnimationUtils = {
 
 		}
 
+	},
+
+	clone: function ( source ) {
+
+		var cloneLookup = new Map();
+
+		var clone = source.clone();
+
+		parallelTraverse( source, clone, function ( sourceNode, clonedNode ) {
+
+			cloneLookup.set( sourceNode, clonedNode );
+
+		} );
+
+		source.traverse( function ( sourceMesh ) {
+
+			if ( ! sourceMesh.isSkinnedMesh ) return;
+
+			var sourceBones = sourceMesh.skeleton.bones;
+			var clonedMesh = cloneLookup.get( sourceMesh );
+
+			clonedMesh.skeleton = sourceMesh.skeleton.clone();
+
+			clonedMesh.skeleton.bones = sourceBones.map( function ( bone ) {
+
+				return cloneLookup.get( bone );
```

Comment:
Do you wanna check the existence of bones in the subtree as I mentioned?

Reply:
Done ✅